clean_up.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+set -e
+
+export IGNORE_NETPOL=("acp-deny-all")
+export IGNORE_CONFIGMAP=("bundle")
+export IGNORE_SECRET=("aws" "keycloak-client" "keycloak-user" "passwordhash" "redis" "rotm-notprod-s3" "rotm-session" "s3-bucket")
+
+export kubectl="kubectl --insecure-skip-tls-verify --server=$KUBE_SERVER --namespace=$KUBE_NAMESPACE --token=$KUBE_TOKEN"
+
+$kubectl delete --all deploy
+$kubectl delete --all svc
+$kubectl delete --all ing
+
+for each in $($kubectl get netpol -o jsonpath="{.items[*].metadata.name}");
+do
+  if [[ ! " ${IGNORE_NETPOL[@]} " =~ " ${each} " ]]; then
+    $kubectl delete netpol "$each"
+  fi
+done
+
+for each in $($kubectl get configmap -o jsonpath="{.items[*].metadata.name}");
+do
+  if [[ ! " ${IGNORE_CONFIGMAP[@]} " =~ " ${each} " ]]; then
+    $kubectl delete configmap "$each"
+  fi
+done
+
+for each in $($kubectl get secrets -o jsonpath="{.items[*].metadata.name}");
+do
+  if [[ ! " ${IGNORE_SECRET[@]} " =~ " ${each} " ]]; then
+    if [[ $each != *"default-token"* ]]; then
+      $kubectl delete secret "$each"
+    fi
+  fi
+done
```

Comment:
Is this a new thing?

Reply:
yes and no, already deployed in 5 other services. It's part of the cron job to tear down branch environments once a week to ensure we don't have too much running and reduce costs

Reply:
nice
what triggers it to run once a week